client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -169,18 +167,21 @@ class MemberProfilePage extends Component {
                 <div className="MemberProfilePage__bio-container">
 
 
-                  {this.state.protectedProfileData !== undefined && this.props.userid !== undefined ? 
-                  <Bio
-                    userid={this.props.userid}
-                    type="default"
-                    first_name={this.state.protectedProfileData.first_name} last_name={this.state.protectedProfileData.last_name}
-                    // name={`${this.props.profile.firstname} ${this.props.profile.lastname}` }
-                    bio={this.state.protectedProfileData.bio}
-                    hashtags={people[0].hashtags}
-                    isCurrentUser={true}
-                    onClick={console.log("Hello")}
-                  />
-                  : '' }
+                  {this.state.protectedProfileData !== undefined && this.props.userid !== undefined ?
+                    <Bio
+                      userid={this.props.userid}
+                      type="default"
+                      first_name={this.state.protectedProfileData.first_name} last_name={this.state.protectedProfileData.last_name}
+                      // name={`${this.props.profile.firstname} ${this.props.profile.lastname}` }
+                      bio={this.state.protectedProfileData.bio}
+                      hashtags={people[0].hashtags}
+                      isCurrentUser={true}
+                      onClick={console.log("Hello")}
+                      instagram={people[0].instagram}
+                      pinterest={people[0].pinterest}
+                      facebook={people[0].facebook}
```

Comment:
These links aren't working for some reason. The URL it creates is like `http://localhost:3000/www.instagram.com`

Reply:
Also looks like it doesn't include the user's social media account. We only want to display that if they have that account on CoTripper and we want them to be able to add that account